index.ts

```diff
@@ -126,7 +126,7 @@ export async function executeScript({
 				throw new Error('chrome.scripting does not support injecting strings of `code`');
 			}
 
-			void chrome.scripting.executeScript({
+			void chromeP.scripting.executeScript({
```

Comment:
Hello, thanks for the PR, but this changes nothing on how code is executed. The only difference is _maybe_ how an error is displayed in the console

Reply:
Thanks for the info and sorry for the trouble.